frontend/src/app/events/pages/event-create/event-create.page.html

```diff
@@ -18,26 +18,39 @@
       <ion-label> Event Title : </ion-label>
       <ion-input type="text" value="" max="150" formControlName="name" class="inputStyle"></ion-input>
     </ion-item>
+    <ion-label floating class="notify" *ngIf="!this.form.get('name').valid">The event name can include characters and space.</ion-label>
```

Comment:
Let's add **dirty** control as well. We don't want to show errors when the user didn't entered anything. Can you check whether **fields are dirty as well** in `*ngIf` statement.

Reply:
Done can you check